src/axom/sidre/core/Group.hpp

```diff
@@ -1120,7 +1120,7 @@ class Group
    *
    * \sa View::isEquivalentTo()
    */
-  bool isEquivalentTo(const Group* other) const;
+  bool isEquivalentTo(const Group* other, bool checkName = true) const;
```

Comment:
Please update the doxygen comment to reflect (and describe) the new parameter.

Reply:
done